docs/API.md

```diff
@@ -974,9 +986,11 @@ An object with the following properties:
 
 #### `"result"`
 
-* `"creation_time"`: A [*timestamp*][Timestamp]. The time at which the *test*
-   was enqueued (in UTC).
-* `"id"`: **Deprecated** (will be removed in v2022.2). An integer.
+* `"creation_time"`: **Deprecated** (planned removal: v2022.2). A [*deprecated timestamp*][Timestamp deprecated].
+  The time at which the *test* was created (in UTC).
+* `"created_at"`: A [*timestamp*][Timestamp]. The time at which the *test* was created
+  (in UTC).
```

Comment:
To have less parentheses and also to move UTC to its natural place "time" I suggest:
```
The time in UTC at which the *test* was created.
```
2 places above and 2 places below.